Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -476,7 +476,7 @@ RefPtr<Gfx::Bitmap> Bitmap::scaled(float sx, float sy) const
 
 RefPtr<Gfx::Bitmap> Bitmap::cropped(Gfx::IntRect crop) const
 {
-    auto new_bitmap = Gfx::Bitmap::create(format(), { crop.width(), crop.height() }, 1);
+    static auto new_bitmap = Gfx::Bitmap::create(format(), { crop.width(), crop.height() }, 1);
```

Comment:
This is not necessary, `new_bitmap` is a RefPtr which will be copied on return.

Reply:
Reverted the change.